tests/core/structured_vespa_index/test_structured_vespa_index.py

```diff
@@ -107,7 +107,8 @@ def test_to_vespa_document_standardMarqoDoc_successful(self):
             'id': 'my_id',
             'fields': {
                 common.FIELD_ID: 'my_id',
-                common.FIELD_SCORE_MODIFIERS: {'price': 100.0, 'rank': 1},
+                common.FIELD_SCORE_MODIFIERS_DOUBLE_LONG: {'rank': 1},
```

Comment:
Please update these tests to include the new data types. You should be able to extend existing cases rather than add new ones

Reply:
Done